Week3/More Javascript/script.js

```diff
@@ -0,0 +1,148 @@
+//1
+
+function info(a, b, c) {
+    console.log(a + b + c)
+}
+info(1, 5, 34);
+
+//2
+
+function colorCar(z) {
+    print = `a ${z} car`
+    console.log(print);
+}
+colorCar("red");
+
+//3
+
+let obj = {
+    name: "Ahmet Husrev",
+    surname: "Semiz",
+    sex: "Male",
+};
+
+function objPrint(obj) {
+
+    console.log(obj);
+}
+objPrint(obj);
+
+//4
+
+function vehicleType(a, b) {
```

Comment:
Same here, do you know  a better name for `a` and `b`? Because now it's really difficult to know what they mean.

Reply:
Got it. :)